{
  "schema_version": "1.4.0",
  "id": "GHSA-67rx-hj63-9h2v",
  "modified": "2023-01-23T18:30:19Z",
  "published": "2023-01-13T03:30:18Z",
  "aliases": [
    "CVE-2022-48256"
  ],
  "details": "Technitium DNS Server before 10.0 allows a self-CNAME denial-of-service attack in which a CNAME loop causes an answer to contain hundreds of records.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
    }
  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2022-48256"
    },
    {
      "type": "WEB",
      "url": "https://github.com/TechnitiumSoftware/DnsServer/blob/master/CHANGELOG.md#version-100"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-835"
    ],
    "severity": "HIGH",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2023-01-13T01:15:00Z"
  }
}